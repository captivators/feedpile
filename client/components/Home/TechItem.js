import React from 'react';

const style = {
  logos: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    boxSizing: 'border-box',
    width: '18%',
    maxWidth: '150px',
    margin: '2px 10px'
  }
}

const TechItem = (props) => {
  return (
    <a style={style.logos} href={ props.tech.link }><img src={ props.tech.image }/></a>
  )
}

export default TechItem;
