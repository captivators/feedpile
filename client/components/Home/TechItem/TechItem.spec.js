import React from 'react';
import { shallow, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import TechItem from './TechItem'

test('Landingpage: TechItem take a snapshot', () => {
  const techItem = {
    name: "React",
    link: "https://facebook.github.io/react/",
    image: "https://s27.postimg.org/ud14uv8v7/logo-578x270.png"
  }

  const component = shallow(
      <TechItem tech={ techItem } />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});