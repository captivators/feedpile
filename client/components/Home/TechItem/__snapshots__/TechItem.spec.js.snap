// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landingpage: TechItem take a snapshot 1`] = `
<div
  style={
    Object {
      "margin": "auto",
      "maxWidth": "150px",
      "padding": "20px",
      "width": "50%",
    }
  }
>
  <a
    className="tech-logo"
    href="https://facebook.github.io/react/"
  >
    <img
      src="https://s27.postimg.org/ud14uv8v7/logo-578x270.png"
    />
  </a>
</div>
`;
